### ISSUE TITLE
[Android][Accessibility] [On input validation failure focus does not go to the first invalid input field when the first invalid field is date/time/compact choiceset]

### ISSUE THREAD

**Issue by dipja** [microsoft/AdaptiveCards#5020]
# Platform

What platform is your issue or question related to? (Delete other platforms).

- [ ] Android

# Author or host

Microsoft Teams

# Version of SDK

2.3.0

# Details
On input validation failure focus does not go to the first invalid input field when the first invalid input field is date/time/compact choiceset.

**Comment by shalinijoshi19** [comment]
@almedina-ms to help look into this specifically around compact choiceset elements.
